web/thesauruses/powershell/7/control_structures.json

```diff
@@ -0,0 +1,128 @@
+{
+  "meta": {
+    "language": "powershell",
+    "language_name": "Powershell",
+    "structure": "control_structures",
+    "language_version": "7"
+  },
+  "concepts": {
+    "if_conditional": {
+      "name": "If conditional",
+      "code": [
+        "if (condition) { ... }"
```

Comment:
Since `...` can mean things in different languages, can you either make that a comment ('like `# ...`) or just leave it empty (which is often valid in languages anyway)?

Same for others with `...`